pkg/server/server_test.go

```diff
@@ -987,6 +987,36 @@ var _ = Describe("ApplicationsServer", func() {
 			Expect(err.Error()).To(ContainSubstring(e.Error()))
 		})
 	})
+	DescribeTable("ValidateProviderToken", func(provider pb.GitProvider, ctx context.Context, errResponse error, expectedCode codes.Code, valid bool) {
+		glAuthClient.ValidateTokenReturns(errResponse)
+		ghAuthClient.ValidateTokenReturns(errResponse)
```

Comment:
I thought about this; I was a little worried this would open things up to a bug where we may be choosing the wrong provider, but some other error may be thrown.

In the case of these table tests, it was just a matter of ensuring both clients had the same behavior, so this made sense.

Reply:
I think I’m with you on the “eh, it’s probably ok” front. If we decide to make the provider selection anything more complex than a single `switch`, we can always revisit it then.